spec/javascripts/govuk-component/option-select-spec.js

```diff
@@ -0,0 +1,281 @@
+describe('GOVUK.OptionSelect', function() {
+
+  var $optionSelectHTML, optionSelect;
+
+  beforeEach(function(){
+
+    optionSelectFixture = '<div class="govuk-option-select" tabindex="0">'+
```

Comment:
This is probably outside the scope of this PR but could the HTML be loaded from an external file? A step beyond that could be to generate it by putting some data into app/views/govuk_component/option_select.raw.html.erb
Just a suggestion, feel free to ignore.. :)